Polygon_mesh_processing/examples/Polygon_mesh_processing/CMakeLists.txt

```
@@ -152,3 +152,14 @@ if(TARGET CGAL::Ceres_support)
 else()
   message(STATUS "NOTICE: The example 'mesh_smoothing_example' uses the Ceres library, and will not be compiled.")
 endif()
+
+
+find_path(CY_INCLUDE "cySampleElim.h" )
+
+if(NOT CY_INCLUDE)
+  message( STATUS "poisson_eliminate needs the CY library")
+else()
+  create_single_source_cgal_program("poisson_eliminate.cpp")
+  target_compile_definitions( poisson_eliminate CGAL_USE_CY )
+  target_include_directories( poisson_eliminate AFTER PRIVATE ${CY_INCLUDE} )
+endif()
```

That message, and the variable `CY_INCLUDE`, will be seen in the CGAL users. I suggest better names.

```suggestion
find_path(CYCODEBASE_INCLUDE "cySampleElim.h" DOC "Path to the cyCodeBase directory (https://github.com/cemyuksel/cyCodeBase)")

if(NOT CYCODEBASE_INCLUDE)
  message( STATUS "poisson_eliminate needs the cyCodeBase library (https://github.com/cemyuksel/cyCodeBase)")
else()
  create_single_source_cgal_program("poisson_eliminate.cpp")
  target_compile_definitions( poisson_eliminate CGAL_USE_CY )
  target_include_directories( poisson_eliminate AFTER PRIVATE ${CYCODEBASE_INCLUDE} )
endif()
```